cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,34 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Results Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <radiogroup class="list-fields" aria-labelledby="show_correctness_label">
```

Comment:
This should be a div with role="radiogroup". https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html

Hopefully that will fix the issue I'm seeing with VoiceOver where it incorrectly says that there are 6 elements in the group (instead of 3).